api/v1alpha3/tags.go

```diff
@@ -118,6 +118,12 @@ const (
 
 	// CommonRole describes the value for the common role
 	CommonRole = "common"
+
+	// TagsLastAppliedAnnotation is the key for the machine object annotation
```

Comment:
just noticed need to fix the comment to match the name here

Reply:
missed that one